cnxarchive/tests/data/search_results.json

```diff
@@ -1,397 +1 @@
"surname": "Physics", "suffix": null, "firstname": "College", "title": null, "fullname": "OSC Physics Maintainer", "id": "cnxcap"}, {"surname": null, "suffix": null, "firstname": "OpenStax College", "title": null, "fullname": "OpenStax College", "id": "OpenStaxCollege"}]}, "limits": [{"tag": "pubYear", "values": [{"count": 3, "value": "2013"}]}, {"tag": "authorID", "values": [{"count": 3, "index": 1, "value": "OpenStaxCollege"}, {"count": 1, "index": 0, "value": "cnxcap"}]}, {"tag": "type", "values": [{"count": 2, "value": "application/vnd.org.cnx.collection"}, {"count": 1, "value": "application/vnd.org.cnx.module"}]}, {"tag": "keyword", "values": [{"count": 1, "value": "ac circuits"}, {"count": 1, "value": "atomic physics"}, {"count": 1, "value": "bioelectricity"}, {"count": 1, "value": "biological and medical applications"}, {"count": 1, "value": "circuits"}, {"count": 2, "value": "college physics"}, {"count": 1, "value": "collisions"}, {"count": 1, "value": "dc instruments"}, {"count": 1, "value": "drag"}, {"count": 1, "value": "elasticity"}, {"count": 1, "value": "electric charge and electric field"}, {"count": 1, "value": "electric current"}, {"count": 1, "value": "electric potential"}, {"count": 1, "value": "electrical technologies"}, {"count": 1, "value": "electromagnetic induction"}, {"count": 1, "value": "electromagnetic waves"}, {"count": 1, "value": "energy"}, {"count": 1, "value": "fluid dynamics"}, {"count": 1, "value": "fluid statics"}, {"count": 1, "value": "forces"}, {"count": 1, "value": "friction"}, {"count": 1, "value": "frontiers of physics"}, {"count": 1, "value": "gas laws"}, {"count": 1, "value": "geometric optics"}, {"count": 1, "value": "heat and transfer methods"}, {"count": 1, "value": "introduction"}, {"count": 1, "value": "kinematics"}, {"count": 1, "value": "kinetic theory"}, {"count": 1, "value": "linear momentum"}, {"count": 1, "value": "magnetism"}, {"count": 1, "value": "medical applications of nuclear physics"}, {"count": 1, "value": "Newton\u2019s Laws of Motion"}, {"count": 1, "value": "Ohm\u2019s Law"}, {"count": 1, "value": "oscillatory motion and waves"}, {"count": 1, "value": "particle physics"}, {"count": 2, "value": "physics"}, {"count": 1, "value": "physics of hearing"}, {"count": 1, "value": "quantum physics"}, {"count": 1, "value": "radioactivity and nuclear physics"}, {"count": 1, "value": "resistance"}, {"count": 1, "value": "rotational motion and angular momentum"}, {"count": 1, "value": "special relativity"}, {"count": 1, "value": "statics and torque"}, {"count": 1, "value": "temperature"}, {"count": 1, "value": "thermodynamics"}, {"count": 1, "value": "uniform circular motion and gravitation"}, {"count": 1, "value": "vision and optical instruments"}, {"count": 1, "value": "wave optics"}, {"count": 1, "value": "work"}]}, {"tag": "subject", "values": [{"count": 2, "value": "Mathematics and Statistics"}, {"count": 1, "value": "OpenStax Featured"}, {"count": 1, "value": "Science and Technology"}]}]}}
```

Comment:
Is it possible for you to redo this change so it formats the json like the original file?